odl/contrib/pytorch/functions.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Predefined functions implementing custom automatic differentiation.
```

Comment:
Empty file?

Reply:
Well I moved around code until I fully understood what `Function` and `Module` objects are there for. But then I had compiled some explaining text that I found helpful.
That said, I could just add the documentation to the `operator.py` file, or the other way around, add `TorchOperator` to `functions.py`.